rootfs/etc/profile.d/prompt.sh

```diff
@@ -91,6 +91,15 @@ function geodesic_prompt() {
 		;;
 	esac
 
+	local level_prompt
+	case $SHLVL in
```

Comment:
Hah! neat idea. Have you seen this elsewhere? The `.` will be kind of odd, no?

Reply:
Of course Geodesic is filled with contex-dependent prompt information, but I have never been in a situation like this where I am routinely in subshells but don't know how deep I am, so no, I've never seen this particular feature before.